admin_guide/high_availability.adoc

```diff
@@ -580,6 +588,54 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
 
+[[admin-guide-high-availability-configuring-more-than-254]]
+=== Configuring IP failover for more than 254 addresses
+
+By default, IP failover management is limited to 254 VIP addresses. You can
```

Comment:
IP failover management is limited to 254 groups.  By default we assign one IP address to each group.  You can change this so multiple IP addresses are in each group by using ...